### ISSUE TITLE
Possible bugfix

### ISSUE THREAD

**Issue by lmmx** [lmmx/emoji-liif#13]
Unless I’m mistaken the loss is not rescaled to [-1,1] and instead only reduced to [-0.5,0.5] from [0,1]:

https://github.com/lmmx/emoji-liif/blob/7a502df7de190bbb07b7a4caac61c1e7d8d13e17/reestimate_leaf_sr_transparency.py#L243

There is a missing multiplication by 2, which explains why the resulting recomposited values did not remove the loss.

Hopefully fixing this will also achieve zero loss (or near enough).

Next, don’t forget to do the positive loss mask when finishing the job with other background shades

**Comment by lmmx** [comment]
While I’m at it (check all of these):

https://github.com/lmmx/emoji-liif/blob/7a502df7de190bbb07b7a4caac61c1e7d8d13e17/reestimate_leaf_sr_transparency.py#L164-L169

> Replace: [skimage.img_as_ubyte](https://scikit-image.org/docs/dev/api/skimage.html#skimage.img_as_ubyte) (import from `skimage.util`)

Also `sklearn.normalize`

```py
img_as_ubyte(normalize(scaled_preproc_img_sub, norm=“max”)).astype(int)
img_as_ubyte(normalize(scaled_source_img_sub, norm=“max”)).astype(int)
img_as_ubyte(normalize(scaled_source_img_sub_alpha, norm=“max”)).astype(int)
```

> Edit: `normalize` will not work here, requires 2D

Note that this will change the type from int to uint unless the astype is added

https://github.com/lmmx/emoji-liif/blob/7a502df7de190bbb07b7a4caac61c1e7d8d13e17/reestimate_leaf_sr_transparency.py#L170

unneeded
